Library/Homebrew/formula_auditor.rb

```diff
@@ -315,7 +315,29 @@ def audit_deps
 
     def audit_conflicts
       formula.conflicts.each do |c|
-        Formulary.factory(c.name)
+        conflicting_formula = Formulary.factory(c.name)
+        problem "Formula should not conflict with itself" if formula == conflicting_formula
+
+        next unless @core_tap
+
+        if CoreTap.instance.formula_renames.key?(c.name) || Formula.aliases.include?(c.name)
+          problem "Formula conflict should be declared using " \
+                  "canonical name (#{conflicting_formula.name}) instead of #{c.name}"
+        end
+
+        rev_conflict_found = false
```

Comment:
What is `rev` here? `reverse`? If so: can you use:
```suggestion
        reverse_conflict_found = false
```